src/Onboarding/Setup/PageView.php

```diff
@@ -139,7 +141,7 @@ public function stripeConnectURL()
                 'website_url' => get_bloginfo('url'),
                 'give_stripe_connected' => '0',
             ],
-            esc_url_raw('https://connect.givewp.com/stripe/connect.php')
+            esc_url_raw(give(ConnectClient::class)->getApiUrl('stripe/connect.php'))
```

Comment:
This class is already using dependency injection in the constructor. Use that instead of retrieving it this way.

Reply:
Resolved in b870f753d05dfb509090e716c9810e4640404030